lib/screens/ol_crowding/dynamic_supervisor.ex

```diff
@@ -48,6 +48,7 @@ defmodule Screens.OlCrowding.DynamicSupervisor do
       restart: :transient
     }
 
-    DynamicSupervisor.start_child(__MODULE__, spec)
+    {:ok, child_pid} = DynamicSupervisor.start_child(__MODULE__, spec)
```

Comment:
In order to avoid crashing the app when one of these fails to start, can this be handled in a `case` instead and if the result is anything but `{:ok, child_pid}` we can just fail silently (or with a log, up to you on that)?

Reply:
mmmm mmhmm 👍 👍